skopt/tests/test_common.py

```diff
@@ -81,22 +83,23 @@ def test_minimizer_api():
     # and does not fit any models
     call_single = lambda res: res.x
     call_list = [call_single, check_result_callable]
+    n_calls = 7
 
     for verbose, call in product([True, False], [call_single, call_list]):
         result = dummy_minimize(branin, [(-5.0, 10.0), (0.0, 15.0)],
-                                n_calls=7, random_state=1,
+                                n_calls=n_calls, random_state=1,
                                 verbose=verbose, callback=call)
 
         assert(result.models is None)
-        yield (check_minimizer_api, result)
-        yield (check_minimizer_bounds, result)
+        yield (check_minimizer_api, result, n_calls)
+        yield (check_minimizer_bounds, result, n_calls)
         assert_raise_message(ValueError,
                              "return a scalar",
                              dummy_minimize, lambda x: x, [[-5, 10]])
 
         n_calls = 7
         n_random_starts = 3
-        n_models = n_calls - n_random_starts
+        n_models = n_calls - n_random_starts + 1
```

Comment:
Seems we can avoid this by doing `fit=False` at the last iteration (which is the current behaviour)?